docs/concepts/submitting-tasks-jobs-to-functions.md

```diff
@@ -17,8 +17,13 @@ Functions can host multiple methods (handlers). You can set the default handler
  need to specify which handler you intend to call in the run command. 
 
 You can pass `parameters` (arguments) or data `inputs` (such as datasets, feature-vectors, models, or files) to the functions through the `run` method.
-- Inside the function you can access the parameters/inputs by simply adding them as parameters to the function, or you can get them from the context object (using `get_param()` and ` get_input()`).
-- Various data objects (files, tables, models, etc.) are passed to the function as data item objects. You can pass data objects using the 
+
+* **Parameters** (`params`) are meant for basic python objects that are serializable, meaning they can be parsed from text without special handling. So, passing `int`, `float`, `str` and `dict`, `list` are all possible using `params`. MLRun takes the parameter and assigns it to the relevant handler parameter by name.
```

Comment:
"basic python objects that are serializable" is how I explain it, not sure it's the most accurate or how it should be painted (@gilad-shaham let us know your opinion)

Reply:
changed to Parameters (`params`) are meant for basic python objects that can be parsed from text without special handling